tests/functional/pages/surveys/census/communal/establishment-type.page.js

```diff
@@ -33,6 +33,11 @@ class EstablishmentTypePage extends MultipleChoiceWithOtherPage {
     return this
   }
 
+  setEstablishmentTypeAnswerOtherOtherText(value) {
```

Comment:
it uses the name of the radio box suffixed with OtherText if the radio option isn't Other would it still make sense?